examples/backups/clickhouse/run-all.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Run every numbered step in order. Stops on the first failure.
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+for step in 01 02 03 04 05 06 07; do
+    script=$(ls "$SCRIPT_DIR"/${step}-*.sh 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)